pkgs/by-name/pq/pqos-wrapper/package.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, intel-cmt-cat
+, fetchFromGitLab
+, python3
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "pqos-wrapper";
+  version = "unstable-2022-01-31";
+
+  src = fetchFromGitLab {
+    group = "sosy-lab";
+    owner = "software";
+    repo = pname;
+    rev = "ce816497a07dcb4b931652b98359e4601a292b15";
+    hash = "sha256-SaYr6lVucpJjVtGgxRbDGYbOoBwdfEDVKtvD+M1L0o4=";
+  };
+
+  makeWrapperArgs = [ "--prefix LD_LIBRARY_PATH : ${lib.getLib intel-cmt-cat}/lib" ];
```

Maybe this would be better?

```suggestion
  makeWrapperArgs = [ "--prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ intel-cmt-cat ]}" ];
```